Sources/AppStoreConnectCLI/Model/API/Sales.Filter+ExpressibleByArgument.swift

```diff
@@ -9,7 +9,7 @@ private typealias Filter = DownloadSalesAndTrendsReports.Filter
 extension Filter.Frequency: ExpressibleByArgument, CustomStringConvertible {
     private typealias AllCases = [Filter.Frequency]
 
-    public static var allCases: AllCases {
+    private static var allCases: AllCases {
```

Comment:
This seems unrelated?

Reply:
> This seems unrelated?

This is for CLI project to be running under Xcode 12. I don't know why it's required but Xcode wouldn't compile without this

Reply:
Right.  Yeah. I noticed earlier actually. Was going to ask.

Reply:
If you changed the type alias to public you could leave the other stuff public.

Reply:
just changed it to public and pushed ✅